appConfig.ts

```diff
@@ -0,0 +1,5 @@
+const appConfig = {
+  DC_URL: process.env.DC_URL,
```

Comment:
You might have to add `API_URL` and `AUTH_TOKEN` here.

Reply:
is this file only used in the production environment?

Reply:
It's used in all environments.